### ISSUE TITLE
Can't log in to deployed Lambda

### ISSUE THREAD

**Issue by ojongerius** [FreeCodeCamp/open-api#144]
I can't log in to stage. Things work fine locally. Guessing it is related to the recent Auth0 changes. @raisedadead / @Bouncey any idea?

Query:

```graphql
query{ getUser(email: "otto@foobar.com") {
  name
}}
```

Response:

```graphql
{
  "data": {
    "getUser": null
  },
  "errors": [
    {
      "message": "You are not authorized, error:0906D06C:PEM routines:PEM_read_bio:no start line",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "getUser"
      ]
    }
  ]
}
```

Headers as follows, $ACCESS_TOKEN the `id_token` from local storage, from https://learn.freecodecamp.org

```
{"Authorization": "Bearer $ACCESS_TOKEN"}
```

Am I doing something wrong here? I thought #131 would have helped but no dice! 😕

**Comment by Bouncey** [comment]
Learn is on the freecodecamp-dev tenant for the time being. I think @raisedadead has a game plan for moving staging over to the dev tenant also?

**Comment by raisedadead** [comment]
Yes, @ojongerius you should have full access to the  `freecodecamp-dev` tenant. Please feel free to use the existing apps and keys or create one, or let me know and I'll create one for you.

As for our production tenant. I'll do the same when we go to production, and add the keys.

**Comment by ojongerius** [comment]
Ok, I'll guess I'll have to roll up my sleeves and do this myself. I'll try to find some time tomorrow.

**Comment by ojongerius** [comment]
I've added cert from the dev tenant to `JWT_CERT_STAGE` in Travis and running an redeployment.

**Comment by ojongerius** [comment]
Hmm that failed with

```
The previous command failed, possibly due to a malformed secure environment variable.
Please be sure to escape special characters such as ' ' and '$'.
For more information, see https://docs.travis-ci.com/user/encryption-keys.
```

Strange, I just took the downloaded PEM file, and converted it with the following code:

```
▶ cat escape-eol.py
import sys
for line in sys.stdin:
    sys.stdout.write(line.strip() + r'\n')
print('')
```

**Comment by ojongerius** [comment]
I'll have another look tomorrow, I guess there is are some other chars in there that Travis does not like.

**Comment by ojongerius** [comment]
This is fixed in #159:

* base64 encoding the certificate, and storing it in JWT_CERT
* decoding it in the code
